Sources/Logging/Logging.swift

```diff
@@ -825,7 +825,7 @@ internal struct StdioOutputStream: TextOutputStream {
                 funlockfile(self.file)
                 #endif
             }
-            _ = fputs(ptr, self.file)
+            _ = fwrite(utf8Bytes.baseAddress!, 1, utf8Bytes.count, self.file)
             if case .always = self.flushMode {
                 self.flush()
             }
```

Comment:
mind adding a `!` to the line below this one (github doesn't let me comment there)? We need to make sure that the `withContiguousStorageIfAvailable` actually worked. It will always work (because your `contiguousUTF8` is correct) but the compiler can't prove this. And just to prevent mistakes by refactorings in the future, we should make sure this actually works.